ui/partials/main/thActiveFiltersBar.html

```diff
@@ -4,17 +4,45 @@
   <span><b>Active Filters</b></span>
 
   <span class="btn btn-sm"
-        ng-show="fromChange().length > 0"
+        ng-show="search()['fromchange'].length > 0"

```

Comment:
It seems like this block could be replaced with an ``ng-repeat`` iterating over an object with the specific pieces for each.  But this list isn't that long, so I'll leave that to your call.  :)  If we add many more, it might be nice to switch to an ``ng-repeat``, though.